cmd/tk/workflow.go

```diff
@@ -88,58 +72,31 @@ func diffCmd() *cobra.Command {
 	)
 
 	cmd.Run = func(cmd *cobra.Command, args []string) {
-		if kube == nil {
-			log.Fatalln(kubernetes.ErrorMissingConfig{Verb: "diff"})
-		}
-
-		raw, err := evalDict(args[0])
+		changes, err := tanka.Diff(args[0],
+			tanka.WithTargets(stringsToRegexps(vars.targets)...),
+			tanka.WithDiffStrategy(*diffStrategy),
+			tanka.WithDiffSummarize(*summarize),
+		)
 		if err != nil {
-			log.Fatalln("Evaluating jsonnet:", err)
-		}
-
-		if kube.Spec.DiffStrategy == "" {
-			kube.Spec.DiffStrategy = *diffStrategy
+			log.Fatalln(err)
 		}
 
-		desired, err := kube.Reconcile(raw, stringsToRegexps(vars.targets)...)
-		if err != nil {
-			log.Fatalln("Reconciling:", err)
+		if changes == nil {
+			log.Println("No differences.")
+			return
 		}
 
-		if diff(desired, interactive, kubernetes.DiffOpts{Summarize: *summarize}) {
-			os.Exit(16)
-		}
-		log.Println("No differences.")
-	}
-
-	return cmd
-}
-
-// computes the diff, prints to screen.
-// set `pager` to false to disable the pager.
-// When non-interactive, no paging happens anyways.
-func diff(state []kubernetes.Manifest, pager bool, opts kubernetes.DiffOpts) (changed bool) {
-	changes, err := kube.Diff(state, opts)
-	if err != nil {
-		log.Fatalln("Diffing:", err)
-	}
-
-	if changes == nil {
-		return false
-	}
-
-	if interactive {
-		h := highlight("diff", *changes)
-		if pager {
+		if interactive {
+			h := highlight("diff", *changes)
 			pageln(h)
 		} else {
-			fmt.Println(h)
+			fmt.Println(*changes)
 		}
-	} else {
-		fmt.Println(*changes)
+
+		os.Exit(16)
```

Comment:
Can the `16` be brought out to a package-level constant so this doesn't feel as magic number-y?